.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- This comments are hidden when you submit the issue, so you do not need to remove them! -->
+<!-- Please be sure to check out our [contributing guidelines](https://github.com/sunpy/sunpy/blob/master/CONTRIBUTING.rst). -->
+<!-- Please be sure to check out our [code of conduct](https://github.com/sunpy/sunpy/blob/master/CODE_OF_CONDUCT.rst). -->
+
+<!-- Please just have a quick search on GitHub to see if a similar pull request has already been posted. -->
+<!-- We have old closed pull requests that might provide useful code or ideas that directly tie in with your pull request. -->
+
+<!-- We have several automatic features that run when a pull request is open.
+<!-- They can appear daunting but do not worry about them! -->
+<!-- We have a brief explanation of [them here](http://docs.sunpy.org/en/latest/dev_guide/pr_review_procedure.html#continuous-integration). -->
+
+### Description
+<!-- Provide a general description of what your pull request does. -->
+
+### Closed Issues
+<!-- If the pull request closes any open issues you can add this. -->
+<!-- If you replace <Issue Number> with a number GitHub will automatically link it. -->
+
+Fixes #<Issue Number>
```

Comment:
<!-- If it doesn't, please remove that line. -->

Reply:
Its in the line above.